sessions/http-api-docs-that-work.md

```diff
@@ -0,0 +1,17 @@
+# Building HTTP API Docs that Work
```

Comment:
One concern that I have with this session is that it focuses on putting a lot of work in documenting something we know for a fact that is broken (in many levels) and needs to be completely redesigned. Focusing on docs rather than redesign will ossify it even further.

Reply:
Wait, *what* are you saying is broken? Is the HTTP API broken?

Otherwise, if you mean the docs themselves, I think this *is* primarily a design activity — at least that’s what I see step 2 as (prototyping people’s ideal version of a few different kinds of endpoints). I totally agree that’s what’s important here. I wasn’t intending that this session produce new docs at all. Maybe I wrote the description here poorly.